packages/docusaurus-plugin-google-analytics/src/index.js

```diff
@@ -53,6 +53,7 @@ module.exports = function(context) {
             innerHTML: `
               window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
               ga('create', '${trackingID}', 'auto');
+              ga('config', '${trackingID}', { 'anonymize_ip': ${anonymizeIP} });
```

Comment:
This syntax is for `gtag.js` not for `analytics.js`. See https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization. The current plugin is for `analytics.js`, not `gtag.js`.

`analytics.js` has a different syntax - https://developers.google.com/analytics/devguides/collection/analyticsjs/ip-anonymization